java/native-image/README.md

```diff
@@ -0,0 +1,19 @@
+# Java Native Image Sample Application
+
+## Building
+
+```bash
+pack build applications/native-image --builder paketobuildpacks/builder:tiny
```

Comment:
```suggestion
pack build applications/native-image --builder paketobuildpacks/builder:tiny --env BP_BOOT_NATIVE_IMAGE=
```

Reply:
oops, forget to add the flags. I also needed `BP_BOOT_NATIVE_IMAGE_BUILD_ARGUMENTS="--initialize-at-build-time=org.springframework.boot.context.properties.bind.Bindable"`